app/game/game.js

```diff
@@ -32,40 +66,130 @@ const GAME = (humanUsername, aiUsername) => {
         }
         return null;
     }
-    const enterRoom = (newRoom) => {
-        // If room not available (cause destroyed), cant enter
-        if (AVAILABLE_ROOMS.indexOf(newRoom) === -1) {
-            console.log(`GAME - Cannot enter room ${newRoom}`);
+    //Whether the room exists on the spaceship
+    const validateRoomPos = (x, y) =>{
+        return x >= 0 && x < SHIP_ROWS && y >= 0 && y < SHIP_COLS;
```

Comment:
Ah, I used to do game development during the first 2 years of programming too, this is typical of position validation in gaming. Haven't seen it in web dev much lol